xarray/core/concat.py

```diff
@@ -0,0 +1,339 @@
+from __future__ import absolute_import, division, print_function
```

Comment:
We don't need these `__future__` imports anymore, now that we're Python 3 only :)